dune

```diff
@@ -42,3 +42,5 @@
  (name runtest)
  (package coqide-server)
  (deps test-suite/summary.log))
+
+(dirs (:standard _build_ci))
```

Comment:
Is this spurious or unrelated?

Reply:
This crept in, it is needed in newer Dune versions to have a compositional overlay build.